docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -35,18 +41,33 @@ while the other pods shouldn't tolerate the taint. The taint is removed once the
 
 *Warning:* currently there is no guarantee which node is chosen and which pods are being killed
 in order to schedule critical pods, so if rescheduler is enabled your pods might be occasionally
-killed for this purpose.
+killed for this purpose. Please ensure that rescheduler should not enabled along with priorities & preemptions in default scheduler as it may cause unwanted side-effects.
 
 ## Config
 
-Rescheduler should be enabled by default. It doesn't have any user facing configuration (component config) or API.
+Rescheduler should be enabled by default(unless you enabled priorities & preemption). It doesn't have any user facing configuration (component config) or API.
 Rescheduler may also be disabled.
 
-### Marking add-on as critical
+### Marking pod as critical when using Rescheduler(This will be deprecated).
 
 To be critical an add-on has to run in `kube-system` namespace (configurable via flag) and
```

Comment:
I'd suggest we be consistent when using terms. It will confuse the readers when you say "pod" in one sentence and then say "add-on" in the next... and you meant the same thing.